src/app/api/v1/endpoints/extract_data.py

```diff
@@ -161,21 +152,16 @@ def create_response(coord, srs):
     raise HTTPException(status_code=404, detail="Coordinates not found.")
 
 
-def extract_text(pdf_page: fitz.Page, user_defined_bbox: fitz.Rect) -> ExtractDataResponse:
-    """Extract text from a PDF Document. The text is extracted from the user-defined bounding box.
+def extract_text(text_lines: list[TextLine]) -> ExtractDataResponse:
+    """Extract numbers from a collection of text lines.
```

Comment:
```suggestion
    """Extract text from a collection of text lines.
```
We return extracted text and bbox based form text lines in ExtractTextResponse